tests/subsys/pm/policy_api/testcase.yaml

```diff
@@ -3,7 +3,9 @@
 
 common:
   tags: pm
-  platform_allow: native_posix native_posix_64
```

Comment:
as there are still many legacy testcases in old format, maybe it is better to add a new case, and keep old ones, as I assume the two format will co-exist for some time.